website/blog/2023-01-20-typescript-first.md

```diff
@@ -0,0 +1,101 @@
+---
+title: First-class Support for TypeScript in React Native
+authors: [NickGerleman, lunaleaps]
```

Comment:
@lunaleaps I think you should be the first author, given you wrote like 95% of this 🙂.